src/Pages/Signature/Detail/DetailPage.js

```diff
@@ -0,0 +1,179 @@
+import React, { useState, useEffect, useRef } from "react";
+import { withRouter } from "react-router-dom";
+import axios from "axios";
+import { Tabs, Theme, Headline3 } from "@class101/ui";
+import FloatingBtn from "../../../Components/Nav/FloatingBtn";
+import Curriculum from "./Curriculum";
+import ClassSummary from "./ClassSummary";
+import ProductView from "./ProductView";
+import IntroSection from "./IntroSection";
+import Package from "./Package";
+import Delivery from "./Delivery";
+import Benefit from "./Benefit";
+import Creator from "./Creator";
+import Recommend from "./Recommend";
+import Refund from "./Refund";
+import Nav from "../../../Components/Nav/Nav";
+import Footer from "../../../Components/Footer/Footer";
+import throttle from "../../../Util/throttle";
+import "./DetailPage.scss";
+
+function DetailPage() {
+  const [currentValue, setCurrentValue] = useState("PREVIEW");
+  const [productData, setProductData] = useState([]);
+  const [detailData, setDetailData] = useState([]);
+  const [tabItems, setTabItems] = useState([]);
+  const [reviewItems, setReviewItems] = useState([]);
+  const [curiItems, setCuriItems] = useState([]);
+  const [isFold, setIsFold] = useState(true);
+  const [isCouponClicked, setIsCouponClicked] = useState(false);
+  const focusTarget = useRef([]);
+
+  useEffect(() => {
+    axios.get("http://localhost:3000/data/productPreview.json").then((res) => {
+      setProductData(res.data.data);
+    });
+
+    axios.get("http://localhost:3000/data/tabitems.json").then((res) => {
+      setTabItems(res.data.data);
+      focusTarget.current = new Array(res.data.data.length);
```

Comment:
```suggestion
      focusTarget.current = [res.data.data.length];
```
굳이 new 연산자 사용할 필요 없습니다